ted_sws/core/model/lazy_object.py

```diff
@@ -0,0 +1,43 @@
+import abc
+from typing import Optional, Any
+
+
+class LazyObjectFieldsLoaderABC(abc.ABC):
+
+    @abc.abstractmethod
+    def load_lazy_field(self, source_object: Any, property_field: property) -> Any:
+        """
+
+        :param source_object:
+        :param property_field:
+        :return:
+        """
+
+
+class LazyObject:
+    _lazy_object_fields_loader: LazyObjectFieldsLoaderABC = None
```

Comment:
This smells. We do not have a separation between the DomainModel and the repository.